myokit/_sim/cvodessim.py

```diff
@@ -361,6 +361,37 @@ def default_state_sensitivities(self):
             return [list(x) for x in self._s_default_state]
         return None
 
+    def state_sensitivities(self):
+        """
+        Returns the sensitivities with respect to state variables, or
+        ``None`` if sensitivities are not enabled.
+        """
+        if self._sensitivities:
+            return [list(x) for x in self._s_state]
+        return None
+
+    def set_state_sensitivities(self, s_values):
+        """
+        Sets the sensitivities with respect to state variables if
+        sensitivities are enabled
+
+        The N x M list of lists 's_values', defines the sensitivities
+        dz/dx for the N state variables and M independent variables
+        """
+        if self._sensitivities:
+            self._s_state = s_values
```

Comment:
This is potentially dangerous: because `s_values` is a list or array or some "non-atomic" type, it can be modified by anyone who has a reference. So we might make some decisions based on its contents, and then the user can still modify it, and our internals get messed up! So copy before storing.